build_docs.pl

```diff
@@ -770,7 +783,9 @@ sub usage {
           --skiplinkcheck   Omit the step that checks for broken links
           --linkcheckonly   Skips the documentation builds. Checks links only.
           --rebuild         Rebuild all branches of every book regardless of what has changed
-          --no_fetch        Skip fetching updates from source repos
+          --keep_hash       Build docs from the same commit hash as last time
+          --sub_dir         Use a directory as a branch of some repo
+                            (eg --sub_dir elasticsearch:master:~/Code/elasticsearch)
```

Comment:
I picked this format because it feels right for CI. It is a bit wordy for folks trying to do it locally but, like I say in the commit message, I don't think this is something we'll run locally too much. At least at first....

Reply:
It's fine by me, but aren't hyphens more conventional in longopts than underscores?

Reply:
Oh, I think we've had that conversation. NVM. :(